client/src/index.js

```diff
@@ -2,11 +2,18 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import Home from "./components/Home";
-import "semantic-ui-css/semantic.min.css";
```

Comment:
This line is deleted here and added again at line 8. Try not to move lines about unnecessarily as it makes it harder to read the commit changes that are relevant to the task.

Reply:
Noted